LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenX3.cs

```diff
@@ -0,0 +1,163 @@
+﻿using HidSharp;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace LibreHardwareMonitor.Hardware.Controller.Nzxt
+{
+    /**
+     * Support for the Kraken X3 devices from NZXT
+     */
+    internal sealed class KrakenX3 : Hardware
+    {
+

```

Comment:
Can you clean up the formatting of this? Auto format should take care of most of it.

Reply:
I am using the standard Visual Studio auto format, and nothing changes. Are you perhaps referring to the Resharper version?

I did update field names and sorted them alphabetically, not sure what to do about the static members though, they do look horribly out of place at the top of the class.

Reply:
Which VS do you use? VS2019 should pick up the editorconfig.

Reply:
Sorry I was doing it wrong, was only formatting the code, not running the CTRL-K + CTRL-E cleanup...